### PR TITLE
Updating docs and keys.go with minor improvements

### PR DESCRIPTION
**Pull request by dinusha92**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: import-export-cli/docs/apimcli_set.md [hunk 1]
```diff
@@ -19,14 +20,17 @@ apimcli set [flags]
 apimcli set --http-request-timeout 3600 --export-directory /home/user/exported-apis
 apimcli set --http-request-timeout 5000 --export-directory C:\Documents\exported
 apimcli set --http-request-timeout 5000
+apimcli set --mode kubernetes
+apimcli set --mode none
```
Comment (kasvith) [inline import-export-cli/docs/apimcli_set.md]: Shall we rename this to `mode normal` 
Mode `none` won't be intuitive. Changing the argument in cobra will do the job

Comment (dinusha92) [reply import-export-cli/docs/apimcli_set.md]: @kasvith  thanks for the suggestions. I have changed that to "default" as it makes more sense than "none"

Comment (harsha89) [reply import-export-cli/docs/apimcli_set.md]: +1

Comment (kasvith) [reply import-export-cli/docs/apimcli_set.md]: +1 That looks fine

**Pull request merged**